src/backend/marsha/core/models/video.py

```diff
@@ -554,3 +554,86 @@ class Meta:
         db_table = "device"
         verbose_name = _("device")
         verbose_name_plural = _("devices")
+
+
+class SharedLiveMedia(UploadableFileMixin, BaseModel):
+    """Model representing a shared media in a video live."""
+
+    RESOURCE_NAME = "sharedlivemedias"
+
+    video = models.ForeignKey(
+        Video,
+        on_delete=models.CASCADE,
+        related_name="shared_live_medias",
+        verbose_name=_("Video"),
+    )
```

Comment:
from a performances pov, shouldn't this need an index (and more likely and ordered one) since this table is requested either using the pk or this column ?

Reply:
Creating a `ForeignKey` defacto creates an index.

```
marsha=# \d shared_live_media
                     Table "public.shared_live_media"
    Column     |           Type           | Collation | Nullable | Default 
---------------+--------------------------+-----------+----------+---------
 deleted       | timestamp with time zone |           |          | 
 id            | uuid                     |           | not null | 
 created_on    | timestamp with time zone |           | not null | 
 updated_on    | timestamp with time zone |           | not null | 
 uploaded_on   | timestamp with time zone |           |          | 
 upload_state  | character varying(20)    |           | not null | 
 title         | character varying(255)   |           |          | 
 show_download | boolean                  |           | not null | 
 nb_pages      | integer                  |           |          | 
 extension     | character varying(10)    |           |          | 
 video_id      | uuid                     |           | not null | 
Indexes:
    "shared_live_media_pkey" PRIMARY KEY, btree (id)
    "shared_live_media_video_id_700af162" btree (video_id)
Check constraints:
    "shared_live_media_nb_pages_check" CHECK (nb_pages >= 0)
Foreign-key constraints:
    "shared_live_media_video_id_700af162_fk_video_id" FOREIGN KEY (video_id) REFERENCES video(id) DEFERRABLE INITIALLY DEFERRED
```